sycl/test/basic_tests/get_backend.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %t.out
+//
+//==----------------- get_backend.cpp ------------------------==//
+// This is a test of get_backend().
+// Also prints handy info about the system.
+// Do not set SYCL_BE.  We do not want the preferred backend.
+//==----------------------------------------------------------==//
+
+#include <CL/sycl.hpp>
+#include <CL/sycl/backend_types.hpp>
+#include <iostream>
+
+using namespace cl::sycl;
+
+bool check(backend be) {
+  switch (be) {
+  case backend::opencl:
+    return true;
+    break;
```

Comment:
minor comment: no need in 'break' right after 'return'.

Reply:
quite right!  done.